Runtime/Localization/LocalizationUtils.cs

```diff
@@ -13,6 +17,135 @@ public static class LocalizationUtils
     {
         private const string defaultIsoCode = "en";
 
+        /// <summary>
+        /// Finds all available languages for the given config, any additional parameters for the paths
+        /// have to be provided. Keep in mind that only sources which are flagged as DefinesAvailableLanguages are
+        /// used and the result will be a union of all of them.
+        ///
+        /// Note: Only sources based on streaming assets work for now.
```

Comment:
remarks?

Reply:
please use the edit function to do minor thingis like this on your own.